Dockerfile

```diff
@@ -3,9 +3,6 @@ FROM node:current-alpine AS builder
 
 WORKDIR /veritable-ui
 
-# Install base dependencies
-RUN npm install -g npm@10.x.x
```

Comment:
we still want to install latest npm so please revert these

Reply:
Should I switch to `npm install -g npm@latest`?

Reply:
`npm update -g npm` adds 30MB
`npm install -g npm@latest` adds 40MB

Reply:
Leave it as `npm install -g npm@10.x.x` as this makes sure we're using the latest available `npm` which is matched by our `package.json` `engines` section. The extra image size is worth it. 

One additional change I might ask for though is to change the base image from `current-alpine` to `lts-alpine`? That would then also match our package json `engines`

Reply:
290MB final image size